io.github.jeffshee.Hidamari.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "io.github.jeffshee.Hidamari",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "hidamari",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-config/hidamari",
```

Comment:
why do you need this?

Reply:
`xdg-config/hidamari` is the location the app writes its config file, and some temporary files it needs.

Reply:
Isn't that `xdg-config` = `~/.config`? The app writes its config data into `~/.config/hidamari/`

Reply:
@hfiguiere Solved in the last commit.